syntax = "proto3";

/*
목적 - 살아있는지 확인만 하는 말그대로 하트비트의 목적.
 */

//manager
//site
message HeartbeatRequest{
  string license_uuid = 1;
  string name = 2;
}


message HeartbeatResponse{
  string license_uuid = 1;
  string name = 2;
}
